modules/catalog/src/main/java/org/apache/ignite/internal/catalog/commands/CatalogUtils.java

```diff
@@ -138,6 +139,13 @@ public class CatalogUtils {
         ALTER_COLUMN_TYPE_TRANSITIONS.put(ColumnType.DECIMAL, EnumSet.of(ColumnType.DECIMAL));
     }
 
+    public static final List<String> SYSTEM_SCHEMAS = List.of(SYSTEM_SCHEMA_NAME);
+
+    /** System schema names. */
+    static boolean isSystemSchema(String schemaName) {
+        return SYSTEM_SCHEMAS.contains(schemaName);
```

Comment:
Do we want to make schema name case insensitive by default?
If so, then it make sense to transform 'schemaName' to upperCase or lowerCase at first.

Reply:
but it schemas uppercased, check https://github.com/apache/ignite-3/pull/3081/files#diff-afb2cc57b87013df26077689351c7b537c1638946ba0828358b7d3296e9e2c0fR327

Reply:
ok.
Do we allow create schema with name quoted name "'Schema'"?